libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -463,6 +463,12 @@ void AP_SerialManager::init()
                     state[i].uart->set_flow_control(AP_HAL::UARTDriver::FLOW_CONTROL_DISABLE);
                     break;
 
+                case SerialProtocol_Sagetech:
+                    state[i].uart->begin(state[i].baud,
+                                         AP_SERIALMANAGER_SAGETECH_BUFSIZE_RX,
+                                         AP_SERIALMANAGER_SAGETECH_BUFSIZE_TX);
```

Comment:
The driver does its own begin

Reply:
good find. I'll remove it from the driver, the default SerialManager one is fine